app/uk/gov/hmrc/apisubscriptionfields/views/application.scala.txt

```diff
@@ -0,0 +1,43 @@
+@(apiContext: String)#%RAML 1.0
+---
+
+title: API Subscription Fields
+version: 1.0
+protocols: [ HTTPS ]
+baseUri: https://api.service.hmrc.gov.uk/
+
+mediaType: [ application/json ]
+
+uses:
+  sec: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/securitySchemes.raml
+  headers: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/headers.raml
+  annotations: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/annotations.raml
+  types: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/types.raml
+
+/@apiContext:
+  /definition:
+    /context:
+      /{applicationContext}:
```

Comment:
You need to add in`uriParameters` keys to describe each path parameter (here and elsewhere in the RAML)